Library/Formula/lcm.rb

```diff
@@ -0,0 +1,48 @@
+class Lcm < Formula
+  desc "libraries and tools for message passing and data marshalling"
+  homepage "https://lcm-proj.github.io/"
+  url "https://github.com/lcm-proj/lcm/releases/download/v1.3.0/lcm-1.3.0.zip"
+  sha256 "23bce19a4a6dc5df444d811f57796e14b4b0d1886cb3c219f90b72f328f2184e"
+
+  head do
+    # This url can be changed to "https://github.com/lcm-proj/lcm.git"
+    # after PR #48 on lcm is accepted.
+    url "https://github.com/bluesquall/lcm.git", :branch => "simplify-java-Makefile.am"
+
+    depends_on "xz" => :build
+    depends_on "libtool" => :build
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on :java => :recommended
```

Comment:
If anyone else wants to chime in on Mike's comment please do so in the next day or so. Thanks!